### PR TITLE
[Docs] Add PHP Attributes examples

### PR DESCRIPTION
**Pull request by simivar**

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -137,45 +139,91 @@ You can configure global information in the bundle configuration ``documentation
...
+    
+    .. code-block:: php-attributes
+    
+        namespace AppBundle\Controller;
+
+        use AppBundle\Entity\User;
+        use AppBundle\Entity\Reward;
+        use Nelmio\ApiDocBundle\Annotation\Model;
+        use Nelmio\ApiDocBundle\Annotation\Security;
+        use OpenApi\Annotations as OA;
```
Comment (DerManoMann) [inline Resources/doc/index.rst:194]: Attributes are in the `OpenApi\Attributes` namespace.

Comment (simivar) [reply Resources/doc/index.rst]: Fixed.

**Comment by GuilhemN** [comment]
Awesome! Thank you @simivar!

**Comment by simivar** [comment]
@GuilhemN I see it broke in the [General PHP objects](https://symfony.com/bundles/NelmioApiDocBundle/current/index.html#general-php-objects) green block. Do you know if it is fixable?

**Comment by GuilhemN** [comment]
Indeed, I believe there was an additional code block that shouldn't be here, I pushed a fix at https://github.com/nelmio/NelmioApiDocBundle/commit/ed33ee8e4a8841ce2c174a60fe2a7c3df955a53e.
Hopefully this should fix the issue.

**Pull request merged**